dropboxdrivefs/core.py

```diff
@@ -22,10 +22,23 @@ class DropboxDriveFileSystem(AbstractFileSystem):
 
     def __init__(self, token, *args, **storage_options):
         super().__init__(token=token, *args, **storage_options)
-        logging.basicConfig(format="%(levelname)s:%(message)s", level=logging.INFO)
```

Comment:
why did you remove this ?

Reply:
Logging should be configured by the user, not set by the package itself.